src/settings/SettingsView.tsx

```diff
@@ -495,6 +494,31 @@ export default function SettingsView(props: IProps): React.JSX.Element {
                     }
                 />
             </SettingsItem>
+            <SettingsItem
+                name={"Ignored tags"}
+                description={
+                    <div>
+                        <p>Files containing any of these tags will be ignored. When you open a file containing a
```

Comment:
Can you add to the description that everything must start with an `#`?
Or maybe even nicer make it work without `#`s